src/core/algorithms/fd/tane/pfdtane.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "algorithms/fd/pli_based_fd_algorithm.h"
+#include "config/error/type.h"
+#include "enums.h"
+#include "model/table/column_data.h"
+#include "model/table/column_layout_relation_data.h"
+#include "model/table/position_list_index.h"
+#include "tane_common.h"
+
+namespace algos {
+
+class PFDTane : public tane::TaneCommon {
```

Comment:
Can you add some description for the algorithm class? (it can be a small comment like for `Pyro` or a full one like for `HyFD`)